corehq/apps/app_manager/id_strings.py

```diff
@@ -445,6 +445,11 @@ def case_search_description_locale(module):
     return "case_search.m{module.id}.description".format(module=module)
 
 
+@pattern('case_search.m%d')
```

Comment:
Is it intentional that this is the same pattern used by `case_search_locale`?

Reply:
It is the same pattern, but looking at the [app strings for a test app on staging](https://staging.commcarehq.org/a/casesearch/apps/download/2de3760a4c584285ae7a97a36e89fe57/default/app_strings.txt) this is what we would want. I decided to make a separate function since `case_search_locale` wasn't intuitive for what we wanted, but I can see reusing that function as well

Reply:
cool changed here: https://github.com/dimagi/commcare-hq/pull/33261#:~:text=%E2%80%A6-,b31872c,-Merge%20state decided to leave the name as is sense nothing else made _more_ sense in the other contexts it's used in

Reply:
cool, thanks